metricbeat/module/mysql/_meta/config.reference.yml

```diff
@@ -2,6 +2,8 @@
   metricsets:
     - "status"
   #  - "galera_status"
+  #  - performance
+  #  - query
```

Comment:
It could make sense to add in the reference docs some commented out example of the query metricset options.